src/current/_includes/v24.3/physical-replication/show-virtual-cluster-data-state.md

```diff
@@ -4,6 +4,6 @@ State      | Description
 `ready` | A virtual cluster's data is ready for use.
 `replicating` | The replication job has started and is replicating data.
 `replication paused` | The replication job is paused due to an error or a manual request with [`ALTER VIRTUAL CLUSTER ... PAUSE REPLICATION`]({% link {{ page.version.version }}/alter-virtual-cluster.md %}).
-`replication pending cutover` | The replication job is running and the cutover time has been set. Once the the replication reaches the cutover time, the cutover will begin automatically.
-`replication cutting over` | The job has started cutting over. The cutover time can no longer be changed. Once cutover is complete, a virtual cluster will be available for use with [`ALTER VIRTUAL CLUSTER ... START SERVICE SHARED`]({% link {{ page.version.version }}/alter-virtual-cluster.md %}).
+`replication pending failover` | The replication job is running and the failover time has been set. Once the the replication reaches the failover time, the failover will begin automatically.
```

Comment:
@alicia-l2 Do we need to add any kind of backward-incompatible note to the release notes to callout these `state` changes?

Reply:
Checked with @alicia-l2 going to leave this for now.